researcher.md

```diff
@@ -16,4 +32,23 @@ The spreadsheets and SQLite3 database only represent most of the data provided b
 
 Source code for download and presentation of the above database can be found on the [untap](https://github.com/abeconnelly/untap) repository.
 
-Please feel free to email the Harvard Personal Genome Project staff if you have any questions or run into any problems.
+### Where can I get the FASTQ/BAM/VCF/etc. files?
+
+Most genomic data available through the Harvard Personal Genome Project site is only available in the format provided to us by Complete Genomics (CGI) and would need to be converted.
+
+CGI gives us unaligned read information, variant evidence files and CGI-Var files, among others.
```

Comment:
I believe that, for a very large fraction of our genomes, do not have "unaligned read information". As I understand it, we elected to not receive the "MAP" directories in data sets in exchange for a lower price. Description of MAP directory here - http://www.completegenomics.com/documents/DataFileFormats_Standard_Pipeline_2.5.pdf